tutorials/Consistent_Scores.ipynb

```diff
@@ -0,0 +1,452 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate a list of thresholds of interest\n",
+    "thetas = murphy_thetas([fcst1, fcst2], obs, \"expectile\")\n",
+    "\n",
+    "# Calculate the average elementary score for the mean (0.5 expectile) for each threshold theta\n",
+    "ms1 = murphy_score(fcst1, obs, thetas, functional=\"expectile\", alpha=0.5)\n",
+    "ms2 = murphy_score(fcst2, obs, thetas, functional=\"expectile\", alpha=0.5)\n",
+    "\n",
+    "# Rename date variable for plotting\n",
+    "ms1 = ms1.rename({\"total\": \"Mean elementary score\", \"theta\": \"rainfall decision threshold (mm)\"})\n",
+    "ms2 = ms2.rename({\"total\": \"Mean elementary score\", \"theta\": \"rainfall decision threshold (mm)\"})\n",
+    "\n",
+    "# Plot the results\n",
+    "ms1[\"Mean elementary score\"].plot()\n",
+    "ms2[\"Mean elementary score\"].plot()\n",
+    "plt.title(\"fcst1 (blue), fcst2 (orange)\")\n",
+    "plt.suptitle(\"Murphy Score (for mean)\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lower values are better on the Murphy Diagram. Fcst2 (orange) performs far worse at higher rainfall thresholds, so is penalised more heavily due to out weighting function $\\phi''(z) = \\exp ^\\frac{z}{10}$."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 2. A score consistent with a quantile\n",
+    "\n",
+    "Suppose we have a 90th percentile wind speed forecast that we want to evaluate, emphasising performance for higher thresholds. We can use the `consistent_quantile_score` in `scores`.\n",
+    "\n",
+    "Our consistent quantile score is defined as\n",
+    "\n",
+    "$$ S(x, y) = \\begin{cases}(1 - \\alpha)(g(x) - g(y)), & y < x \\\\\\alpha(g(y) - g(x)), & x \\leq y\\end{cases}$$\n",
+    "\n",
+    "where:\n",
+    "\n",
+    "- $x$ is the forecast\n",
+    "- $y$ is the observation\n",
+    "- $\\alpha$ is the quantile level\n",
+    "- $g$ is a nondecreasing function of a single variable\n",
+    "- $S(x,y)$ is the score.\n",
+    "\n",
+    "Now suppose we want to emphasise the performance for higher thresholds, but we don't want our weights to increase exponentially like in rainfall example above. We first define our weighting function as \n",
```

Comment:
Maybe add something about absolute error: that taking alpha0.5 and g(z)=2z (please check the constant!) recoveres absolute loss, with the interpretation that MAE equal weights on each decision threshold since g'(z) is constant.

Reply:
I also think that for both expectiles and quantiles you should make a genera statement about the wieght placed on decision thresholds, namely the weighting function is phi'' for expectiles and g' for quantiles. Only then should you discuss the specific weight for your problem. Otherwise readers may wonder whether you've made a mistake in assuming the weights are second derivatives in the first example and first derivatives in the second